scholia/app/templates/work_cito.html

```diff
@@ -7,7 +7,7 @@
 {{ sparql_to_table('cito-incoming',
     options={
     "linkPrefixes": {
-	     "intention": "../../cito/",
+	     "intention": "/work/" + q + "/cito/",
```

Comment:
@egonw I'm not sure if the `q` is alone or must be enclosed by curly braces because maybe `q` is a jinja's expression

Reply:
I double checked the code and when I run it, it does as expected. The curly brackets _are_ around the `q`, actually, and on line 7. I originally had the curly brackets as `"intention": "/work/{{ q }}/cito/"` but then it ended up encoded in the URL :)